.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -20,8 +20,13 @@ If this is a new visual feature please add a before/after screenshot or gif
 here with e.g. [GifGrabber](http://www.gifgrabber.com/).
 
 # Notify reviewers
-If your pull request involves changes to an existing file, one or more of the
-people that edited before you might be good candidates to review it. Please use
-`git blame <filename>` to determine that and notify them here. Otherwise notify
-everybody in the core team:
-@cBioPortal/core-developers
+Read our [Pull request merging policy](../CONTRIBUTING.md#pull-request-merging-policy). Notify the approprate
+team (remove inappropriate):
+
+@cBioPortal/frontend
+@cBioPortal/backend
+@cBioPortal/devops
```

Comment:
I see, that's a bummer. Looks like only members of the organization can see the teams. Maybe we should add a document then showing the teams.